src/gtfs/mod.rs

```diff
@@ -91,9 +91,9 @@ struct Stop {
     name: String,
     #[serde(default, rename = "stop_desc")]
     desc: String,
-    #[serde(rename = "stop_lon")]
+    #[serde(rename = "stop_lon", deserialize_with = "de_location_trim_with_default")]
```

Comment:
you could use https://docs.rs/csv/1.0.2/csv/struct.ReaderBuilder.html#method.trim

Reply:
agreed with u if we can trim all fields, what do u think @kinnou02?

Reply:
good for me

Reply:
trimming both content and header seems to be a good idea, we know some GTFS have spaces in col names